activities-examples/README.md

```diff
@@ -3,8 +3,8 @@
 This sample shows use cases for Activities:
 
 - [`makeHTTPRequest`](./src/activities/index.ts): Make an external HTTP request in an Activity (using `axios`).
-- [`cancellableFetch`](./src/activities/cancellable-fetch.ts): Make a cancellable HTTP request with [cancellationSignal](https://typescript.temporal.io/api/classes/activity.context/#cancellationsignal).
-- [`doSomethingAsync`](./src/activities/async-completion.ts): Complete an Activity async with [`AsyncCompletionClient`](https://typescript.temporal.io/api/classes/client.AsyncCompletionClient#complete).
+- [`cancellableFetch`](./src/activities/cancellable-fetch.ts): Make a cancellable HTTP request with [cancellationSignal](https://docs.temporal.io/application-development/testing/#cancel-an-activity).
```

Comment:
I don't think you want to change this one?

Reply:
didn't know the link vanished. thanks

Reply:
not sure what you mean.
But https://typescript.temporal.io/api/classes/activity.context/#cancellationsignal should not change to https://docs.temporal.io/application-development/testing/#cancel-an-activity
So this needs to be reverted.

Reply:
Page wasn't found. May have taken on an additional directory, so I'll check on that